### PR TITLE
Refactor nails logger

### PR DESCRIPTION
**Pull request by kkaempf**
On top of https://github.com/MaximilianMeister/nailed/pull/28

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
there are still some references of `get_config` in
* lib/nailed.rb
* lib/nailed/bugzilla.rb
* lib/nailed/jenkins.rb

https://github.com/kkaempf/nailed/commit/0eb11d89959498c82153df96ab70e33205d42458 and https://github.com/kkaempf/nailed/commit/5460ab26e57701a05e958781f28009a074fd2ab4 could be squashed

but in general @kkaempf you have some commits here in this PR as well in #28, on the first look it seems you can close #28, is that correct?

BTW thanks already for the contribution :-)

File: lib/nailed/github.rb [hunk 1]
```diff
@@ -1,14 +1,75 @@
...
+  
+  #
+  # github helpers
+  #
+  def get_org_repos(github_client, org)
+    all_repos = github_client.org_repos(org)
+    all_repos.map(&:name)
+  end
+
+  def fill_db_after_migration(github_client)
```
Comment (MaximilianMeister) [inline lib/nailed/github.rb]: I would also keep this method in `lib/nailed.rb` as it doesn't have only a GitHub context

**Comment by kkaempf** [comment]
Maybe you should merge #28 first and then I rebase #29 on top

File: lib/nailed/github.rb (hunk 1 above)
Comment (kkaempf) [inline lib/nailed/github.rb]: I deliberately moved this to `lib/nailed/github.rb` to give it context. `Nailed.get_org_repos` doesn't tell you which org and which repos. `Nailed::Github.get_org_repos` is much clearer.

Comment (MaximilianMeister) [inline lib/nailed/github.rb]: I was only meaning the `fill_db_after_migration` method. the other ones I fully agree

**Comment by MaximilianMeister** [comment]
> Maybe you should merge #28 first and then I rebase #29 on top

okay, then I will look at #28, and after that we can come back here to discuss

**Comment by MaximilianMeister** [comment]
#28 got merged, so you can rebase now. thanks

**Comment by kkaempf** [comment]
Will open new PR

**Pull request closed without merging**